source/funkin/game/PlayState.hx

```diff
@@ -1260,7 +1260,7 @@ class PlayState extends MusicBeatState
 				if (Conductor.songPosition >= 0)
 					startSong();
 			}
-		} else {
+		} else if (!startingSong && FlxG.sound.music != null) {
```

Comment:
The !startingSong isn't needed due to the if condition above

Reply:
letme fix it

Reply:
done